app/src/main/java/com/framgia/englishconversation/screen/imageDetail/ImageDetailViewModel.java

```diff
@@ -18,8 +22,20 @@
     private TimelineModel mTimelineModel;
     private FragmentManager mManager;
     private Fragment mFragment;
+    private Context mContext;
+    private OnMediaModelItemTouchListener<List<MediaModel>> mListener =
+            new OnMediaModelItemTouchListener<List<MediaModel>>() {
 
-    public ImageDetailViewModel(TimelineModel timelineModel, FragmentManager manager) {
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.